test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] Kindly add the name of this file in the `.gitignore` file, as the requirements say.